### PR TITLE
Foods search paging

### PR DESCRIPTION
**Pull request by joseph4tw**
Added paging to server
Updated client to use paging (just first page and max of 5 to show that it works)
Updated fatSecretClient test

### CODE REVIEW

**Comment by joseph4tw** [comment]
[Food Search:  Implement paging so that a user can browse through the entire search results](https://trello.com/c/H0VhmvDv/22-food-search-implement-paging-so-that-a-user-can-browse-through-the-entire-search-results)

File: client/src/services/foods.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ export const SearchFoods = (query) => {
     const storage = new Storage();
     const token = storage.getAuthToken();
     const config = {};
-    const url = `/api/foods?q=${query}`;
+    const url = `/api/foods?q=${query}&p=0&m=5`;
```
Comment (kevinsa) [inline client/src/services/foods.js]: I see some defaults to 20 in the client for max results, if the client calls the controller with just one param (q=cashews) would the max_results returned be 5 or the FatSecret default of 20?

Comment (kevinsa) [reply client/src/services/foods.js]: I guess query.params.m would not be defined so the max results would be 20...

Comment (joseph4tw) [reply client/src/services/foods.js]: @kevinsa that's right. The defaults live in the FatSecretClient on the server side.

**Pull request merged**